.github/workflows/ci.yml

```
@@ -10,28 +10,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v4.1.1
         with:
           path: tachimawari
 
       - name: Checkout Tachimawari Interfaces
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v4.1.1
         with:
           repository: ichiro-its/tachimawari_interfaces
           path: tachimawari_interfaces
 
       - name: Checkout Kansei Interfaces
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v4.1.1
         with:
           repository: ichiro-its/kansei_interfaces
           path: kansei_interfaces
 
       - name: Checkout Keisan
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v4.1.1
         with:
           repository: ichiro-its/keisan
           path: keisan
 
+      - name: Checkout Jitsuyo
+        uses: actions/checkout@v4.1.1
+        with:
+          repository: ichiro-its/jitsuyo
+          path: jitsuyo
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Install nlohmann/json
+        run: sudo apt update && sudo apt install -y nlohmann-json3-dev
```

Can you try without `sudo apt update`? see if it behaves the same.

```suggestion
        run: sudo apt install -y nlohmann-json3-dev
```